src/calculateRentalCost.js

```diff
@@ -26,9 +26,23 @@
  *
  * @return {number}
  */
+const PRICE_PER_DAY = 40;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM = 3;
+const SHORT_TERM_DISCOUNT = 20;
 
 function calculateRentalCost(days) {
   // write code here
+  let basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    basePrice -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
```

Comment:
4.[checklist] Prefer `if` with `return` over `if else` to simplify later conditions.

Reply:
done